VAMobile/src/components/WaygateWrapper.tsx

```diff
@@ -65,7 +65,7 @@ export const WaygateWrapper: FC<WaygateWrapperProps> = ({ children, waygateName,
             />
           </Box>
           {waygate.appUpdateButton === true && <Button onPress={onUpdateButtonPress} label={t('updateNow')} />}
```

Comment:
Could potentially be simplified moving this to the `primaryButton` prop to AlertWithHaptics.